source/Calamari.Tests/KubernetesFixtures/HelmUpgradeFixture.cs

```diff
@@ -325,6 +325,18 @@ public void AdditionalArgumentsPassed()
             result.AssertOutputMatches("[helm|\\\\helm\"] upgrade (.*) --dry-run");
         }
 
+        [Test]
+        [RequiresNonFreeBSDPlatform]
+        [RequiresNon32BitWindows]
+        [RequiresNonMac]
+        [Category(TestCategory.PlatformAgnostic)]
+        public void WhenTheChartDirectoryVariableIsSet_TheChartAtThatLocationIsUsed()
+        {
+            Variables.Set(Kubernetes.SpecialVariables.Helm.ChartDirectory, "specific/location/for/my/chart");
+            var result = DeployPackage("mychart-with-specific-location-0.3.8.tar.gz");
```

Comment:
For extra points, would it be possible to have a way to create these archives in code so we don't have to check in a binary? Or like 'nah get stuffed mate, this is good enough'?

Reply:
It would be nice, I spent a bunch of time mucking around getting the right archive shape. I'll take a quick look to see if we have something already, else prob won't worry too much.